static/docs/commands-reference/push.md

```diff
@@ -86,11 +86,12 @@ backward through the pipeline to find data files to push.
   are associated with the named stage, and the stages which execute earlier in
   the pipeline.
 
-- `-R`, `--recursive` - the `targets` value is expected to be a directory path.
-  With this option, `dvc pull` determines the files to upload by searching the
-  named directory, and its subdirectories, for DVC files for which to upload
-  data. Along with providing a `target`, or `target` along with `--with-deps`,
-  it is yet another way to limit the scope of DVC files to upload.
+- `-R`, `--recursive` - the `targets` value is expected to be a directory path
+  and also files as well. With this option, `dvc pull` determines the files to
+  upload by searching the named directory, and its subdirectories, for 
+  DVC-files for which to upload data. Along with providing a `target`, or 
+  `target` along with `--with-deps`, it is yet another way to limit the scope 
+  of DVC-files to upload.
```

Comment:
Ideas:
"a directory path and also files as well" -> "path to a file or directory, relative to the current location (e.g. data/models`)";
"named directory, and its subdirectories, for DVC-files for which to upload data." -> "target directory and its subdirectories for DVC-files.";
"`target`, or `target` along with `--with-deps` it is yet another" -> "`target` and/or `--with-deps`, its another";

> Since they seem to repeat in different docs, I would also search for these phrases in all of the md files, to update the language everywhere consistently.